highfidelity/hifi
# MS19052 [CreateApp]: Tooltips

## ~~Note: THIS PR IS DNM until https://github.com/highfidelity/hifi/pull/14172 is merged~~

https://highfidelity.manuscript.com/f/cases/19052

![image](https://user-images.githubusercontent.com/607735/47248150-b36ea580-d408-11e8-8db2-ebd3422b49f7.png)

Tooltips should appear for all properties when hovering their label for atleast a half second.

## QA test
1. Open the create app
2. Create an entity of each different type:
   - Model
   - Shape (Cube or Sphere)
   - Light
   - Text
   - Image
   - Web
   - Zone
   - Particle
   - Material
3. Go through each of the entities their properties in the properties tab, hover over all the properties and make sure that the tooltip is available. **(Note: It is expected that all Start and Finish properties except for Horizontal Angle Start don't have a tooltip. This is because the design will change in RC76)**.
4. Tooltips should **not** be available while in hmd, try hovering over the properties labels and make sure that the tooltips don't show up.
